apps/src/templates/Congrats.jsx

```diff
@@ -6,6 +6,9 @@ import Certificate from './Certificate';
 import StudentsBeyondHoc from './StudentsBeyondHoc';
 import TeachersBeyondHoc from './TeachersBeyondHoc';
 import { tutorialTypes } from './tutorialTypes.js';
+import { Provider } from 'react-redux';
+import { combineReducers, createStore } from 'redux';
+import responsiveRedux from '../code-studio/responsiveRedux';
```

Comment:
Do we expect these?

Reply:
Yes. `ResourceCardResponsiveContainer` needs access to the store already.  This will be consistent with the changes in #19042.